projects/observability/src/shared/components/bar-gauge/bar-gauge.component.ts

```diff
@@ -48,9 +49,10 @@ import {
               [ngClass]="{ 'hide-last-divider': this.nearMaxValue }"
               [style.background]="segment.color"
               [style.width.%]="segment.percentage"
-              htTooltip="{{ segment.label }} : {{ segment.value | htDisplayNumber }}"
+              [htTooltip]="this.segmentsTooltip ? this.segmentsTooltip : plainTooltip"
```

Comment:
Not a big deal so don't change it unless you are making other changes, but would `this.segmentsTooltip ?? plainTooltip` work here?